### PR TITLE
Panacea - Soft Viro Nerf

### PR DESCRIPTION
**Pull request by eeSPee**
## About The Pull Request

Adds panacea, a reagent that boosts your immune system. On overdose, it cures all diseases without making antibodies, but the user takes stamina damage for a while. Only available in CMO locker and purchasable from cargo in a 20000cr crate.

## Why It's Good For The Game

Viro can make diseases that can decimate entire populations. Once that reaches, it becomes quite unstoppable and very few people (CMO, chemists and maybe the Captain) can stop it. Other departments can't really do anything against it. 

This isn't meant to be widely distributed and cure an entire disease. Instead, it is meant to be taken by one person (ie chemist, CMO) so they can treat the disease without getting infected and it becoming a race against time.

With the upcomming Chem Dispenser chip, viro can truly become unstoppable if he takes out the proper people. This is an idea on how to control that.

IMO viro recieved a couple of buffs that put it a bit over the edge. This adds a little bit of counterplay maybe. Just an idea.

## Changelog
:cl:
add: New reagent: panacea, that cures all current diseases on injected.
add: Bio Emergency Crate comes with one injection of panacea, and CMO has one in their locker
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -1596,3 +1596,27 @@
...
+	reagent_state = LIQUID
+	color = "#CC6666"
+	metabolization_rate = 0.75 * REAGENTS_METABOLISM
+	taste_description = "snake oil"
+	process_flags = ORGANIC
+	random_unrestricted = FALSE
+
+/datum/reagent/medicine/panacea/on_mob_life(mob/living/M)
+	if (volume>10)
+		for(var/thing in L.diseases)
```
Comment (utilbird) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: L isn't defined, did you mean M?

```suggestion
		for(var/thing in M.diseases)
```

**Review by utilbird — COMMENTED** [review]
Arghh, remove the spaces between `if`s and their args

**Comment by eeSPee** [comment]
> Arghh, remove the spaces between `if`s and their args

I'm used to C#.

**Comment by leestriter** [comment]
I don't see this being needed. Honestly viruses are not that hard to cure/counter as is, and even a traitor isn't allowed to use the bad ones on mrp unless their goal is shuttle hijack. Most of the time a viro isn't gonna have the time to make the best shit unless its a long round or rng is very forgiving. I've made some gnarly viruses in lrp and most of the time one or two competent people can counter it pretty easily, and if there's a cmo or ai checking on you you will be caught before release. Many people will have on some sort of covering preventing infection at a given time anyway. Along with that the best symptoms already have to be bought at cargo or you get really luckily running around maints. An instant cure to anything isn't a "soft nerf", one cure of the earliest infected can stop any disease before it starts. Also shouldn't be in the biological emergency crate it should be separate, the crate should remain a defensive option. I think the reduced price of the junior epidemiology kit is a good enough nerf already, as it makes it simple to bypass virus stealth and produce vaccines.

**Comment by nikk735** [comment]
I could see this being cool, pretty balanced overall cuz chances are ur gonna get sick again instantly but can be useful in specific situations like a super lethal virus or GBS

**Comment by leestriter** [comment]
You wouldn't get sick again cause you'd put yourself in a full suit with internals, then cure yourself. Then, you just go make the vaccine and everything is good.

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -68,14 +68,16 @@
 /datum/supply_pack/emergency/bio
 	name = "Biological Emergency Crate"
 	desc = "This crate holds 2 full bio suits which will protect you from viruses."
-	cost = 1500
+	cost = 20000
```
Comment (Archanial) [inline code/modules/cargo/packs.dm:71]: This should be documented in changelong.

**Comment by Isy232** [comment]
A competent chemist can mix about any reagent that does not require anything but what is in chemistry in like 2 or 3 minutes.

Viro is powerful, but not OP.

The counterplay is when you hear 'VIRO ROGUE' you slap on that biosuit and internals or rush to make a cure.

We don't need an insta cure magical reagent in the game.

**Comment by DatBoiTim** [comment]
Insta Cure available roundstart(even if in limited quantity)? You out of your mind? I've seen a random person just waltz into chemistry and cure Resistance 13+ viruses in a few minutes. This is in addition to the Holodeck having capacity to generate vaccines which are instant cures and protect from infection if you don't have it.

**Comment by yyzsong** [comment]
cucks sentient diseases. Cringe.

**Comment by ghost** [comment]
Bio suits don't actually do anything, they're purely cosmetic.

**Comment by DatBoiTim** [comment]
> Bio suits don't actually do anything, they're purely cosmetic.

Wait really? Wiki should probably have be updated so the infection spread is accurate.

**Comment by utilbird** [comment]
"soft"

**Comment by eeSPee** [comment]
Closed because it is not necessary.

**Pull request closed without merging**